CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.1.1 (2022-02-07)
```

Comment:
This is not the right release date for the next release. I usually use `TBD` instead until the release is made.

```suggestion
# v1.1.1 (TBD)
```